### PR TITLE
[quant][graphmode][fx] Add support for conv module

### PR DESCRIPTION
**Pull request by jerryzh168**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #43377 [quant][graphmode][fx] Add support for instance_norm
* #43376 [quant][graphmode][fx] Add support for layer_norm
* #43375 [quant][graphmode][fx] Add support for elu
* #43374 [quant][graphmode][fx] Add support for hardswish
* #43373 [quant][graphmode][fx] Add support for mul and mul relu
* #43372 [quant][grapphmode][fx][test][refactor] Refactor quantized add test
* #43335 [quant][graphmode][fx] Add support for batchnorm relu
* #43334 [quant][graphmode][fx] Add support for batchnorm
* #43333 [quant][graphmode][fx] Add support for cat
* #43332 [quant][graphmode][fx] Add support for add relu
* #43331 [quant][graphmode][fx] Add support for add
* #43287 [quant][graphmode][fx] Add support for conv module + relu
* #43286 [quant] Make OP_LIST_TO_FUSER_METHOD public
* **#43285 [quant][graphmode][fx] Add support for conv module**

Summary:
Porting op tests from test_quantize_jit.py

Test Plan:
TestQuantizeFxOps

Reviewers:

Subscribers:

Tasks:

Tags:

Differential Revision: [D23221733](https://our.internmc.facebook.com/intern/diff/D23221733)

### CODE REVIEW

**Review by vkuzo — APPROVED** [review]

File: torch/quantization/fx/quantize.py [hunk 1]
```diff
@@ -213,17 +216,11 @@ def convert(self, quantizer, node, load_arg, debug=False):
...
-                qconv = torch.nn.quantized.Conv2d
-            elif type(self.conv) in [torch.nn.intrinsic.ConvReLU2d,
-                                     torch.nn.intrinsic.qat.ConvReLU2d,
-                                     torch.nn.intrinsic.qat.ConvBnReLU2d]:
-                qconv = torch.nn.intrinsic.quantized.ConvReLU2d
-            else:
-                raise Exception("unhandled conv type:", type(self.conv))
-            quantized = qconv.from_float(self.conv)
+            # TODO: make the mapping configurable?
+            assert type(self.conv) in DEFAULT_MODULE_MAPPING, \
```
Comment (raghuramank100) [inline torch/quantization/fx/quantize.py:220]: This is a looser check than the previous check which was checking for specific module classes. Is this sufficient?

Comment (jerryzh168) [reply torch/quantization/fx/quantize.py:220]: yeah it is equivalent, the check on the left side can be represented by the mapping.
